tests/test_cookiejar.py

```diff
@@ -662,3 +662,20 @@ def test_cookie_not_expired_when_added_after_removal(self) -> None:
         next(iter(dummy_jar))
     assert not dummy_jar.filter_cookies(URL("http://example.com/"))
     dummy_jar.clear()
+
+
+async def test_loose_cookies_types() -> None:
+    jar = CookieJar()
+
+    accepted_types = {
+        'StrBaseCookieTuples': [('str', BaseCookie())],
+        'StrMorselTuples': [('str', Morsel())],
+        'StrStrTuples': [('str', 'str'), ],
+        'StrToBaseCookieMapping': {'str': BaseCookie()},
+        'StrToMorselMapping': {'str': Morsel()},
+        'StrToStrMapping': {'str': 'str'},
+        'BaseCookie[str]': SimpleCookie(),
+    }
+
+    for name, loose_cookies_type in accepted_types.items():
```

Comment:
`name` is not used. So `accepted_types` can be a list instead of a dict.

Alternatively, you can remove a loop and write a sequence of calls:
```python
jar.update_cookies(cookies=[('str', BaseCookie())])
jar.update_cookies(cookies=[('str', Morsel())])
...
```

Reply:
Switched to using list. Thanks